lib/resources/r4/related_person_patch.yaml

```diff
@@ -126,6 +126,45 @@ operations:
         <li>When specifying a `period`, the `start` and `end` fields must include a time component with a timezone.</li>
       </ul>
 
+  - name: add-telecom
+    path: /telecom/-
+    operation: add
+    type: ContactPoint
+    url: https://hl7.org/fhir/R4/patient-definitions.html#Patient.telecom
```

Comment:
should be RelatedPerson.telecom

Reply:
Done:https://github.com/cerner/fhir.cerner.com/pull/559/commits/fc371d3cda17ab389ec751373f0a9a0b98dea62d#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR133

Reply:
I don't think the link is working yet. Please try and test the link to be working after you change.
I think it should be https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson. telecom

Reply:
Updated here and validated link works: https://github.com/cerner/fhir.cerner.com/pull/559/commits/9a5355ede6a1e2988bb7e041faf5224a1ef4acdb#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dL133